main.go

```diff
@@ -91,6 +112,8 @@ func (cfg config) getBuilds(f filter) (builds, error) {
 		return builds{}, fmt.Errorf("invalid response status code: %d\nbody: %s", resp.StatusCode, string(body))
 	}
 
+	log.Printf("%s", body)
```

Comment:
instead of using separate logging for the request I suggest to use httputil.DumpRequest